README.md

```diff
@@ -11,9 +11,9 @@
 [![Translate Pressbooks](https://img.shields.io/badge/dynamic/json.svg?label=translated&url=https%3A%2F%2Ftenpercent.now.sh%2F%3Forganization%3Dpressbooks%26project%3Dpressbooks&query=%24.status&colorB=e05d44&suffix=%25)](https://www.transifex.com/pressbooks/pressbooks/translate/)
 
 
-[Pressbooks](http://pressbooks.org) is a book content management system which
+[Pressbooks](https://pressbooks.org) is a book content management system which
 produces multiple formats: ebooks, webbooks, print-ready [PDF][pdf], and various
-XML flavours. [Pressbooks](http://pressbooks.org) is built on top of
+XML flavours. [Pressbooks](https://pressbooks.org) is built on top of
 [WordPress Multisite](http://codex.wordpress.org/Glossary#Multisite), and makes
```

Comment:
Link deprecated. Can replace with http://codex.wordpress.org/Glossary#Multisite for better user experience